test/event_listeners.test.js

```diff
@@ -0,0 +1,128 @@
+/* eslint-disable import/no-unresolved */
+
+import * as events from '../src/Controllers/event_listeners';
+
+const $ = require('jquery');
```

Comment:
- [ ] Please avoid using require, rather use es6 import syntax. Perhaps you can do this:
`import * as $ from 'jquery'`